safe/gui/tools/batch/scenario_runner.py

```diff
@@ -57,7 +57,8 @@ def completed(flag):
         LOGGER.debug("scenario done")
         dock.analysis_done.disconnect(completed)
 
-    dock.analysis_done.connect(completed)
+    # commenting this since the signal is removed from dock
```

Comment:
Maybe just clean it away completely. I dont think we should be worrying about the dock when using the batch runner...